runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -260,6 +262,25 @@ public void write(IToken token, Object data) throws OverwriteException {
         return fetch(address);
     }
 
+    /**
+     * Read a range of addresses in a predictive way.
+     *
+     * If there is a cache miss, read a range of addresses, otherwise
+     * directly return the requested entry and read is defer.
+     */
+    public @Nonnull ILogData predictiveReadRange(Long address, List<Long> addresses) {
```

Comment:
What is wrong with readCache? Why not rely on its internal mechanism to fetch logData?